src/amberscript/parser.cc

```diff
@@ -995,6 +1019,41 @@ Result Parser::ParseClear() {
   return ValidateEndOfStatement("CLEAR command");
 }
 
+Result Parser::ParseValues(const std::string& name,
+                           const DatumType& type,
+                           std::vector<Value>* values) {
+  assert(values);
+
+  auto token = tokenizer_->NextToken();
+  while (!token->IsEOL() && !token->IsEOS()) {
+    Value v;
+
+    if ((type.IsFloat() || type.IsDouble())) {
+      if (!token->IsInteger() && !token->IsDouble()) {
```

Comment:
IsInteger -> IsFloat

Reply:
The token doesn't have an IsFloat, just IsDouble. We ConvertToDouble a couple of lines down, so if we have an Integer or a Double then we're good to go. If we get anything other then a number then things go wrong.

Reply:
ok.  I was wrong.